homeassistant/components/ps4/media_player.py

```diff
@@ -175,6 +186,23 @@ def _parse_status(self):
         else:
             self._retry += 1
 
+    def _use_saved(self) -> bool:
+        """Return True, Set media attrs if data is locked."""
+        if self._media_content_id in self._games:
+            store = self._games[self._media_content_id]
+
+            # If locked get attributes from file.
+            locked = store.get(ATTR_LOCKED)
```

Comment:
What does locked mean?

Reply:
Locked if true means that we will not retrieve data externally. But rather use the data we saved locally in the file. The data will not change unless unlocked or edited. 

If false we will retrieve data externally as we do currently for status updates, and use the data we get back.

Reply:
What is the use case?

Reply:
Users can edit attributes with text editor, for example, title or image url, retrieved is incorrect. There is some error with searching for these attributes from the external database as items are constantly being updated and there may be differences in the information we get from the device vs. the database. As such the info we get from the device may no longer exist in the database for the name of the media has been changed slightly and so cannot be found.

With this, it gives users the ability to correct data themselves. As of now edited data will be overwritten by the integration with status updates. By providing the lock mechanism, the data will not be changed.